Source/Core/DolphinQt/Config/Graphics/GraphicsWindow.cpp

```diff
@@ -20,18 +20,32 @@
 #include "DolphinQt/Config/Graphics/HacksWidget.h"
 #include "DolphinQt/MainWindow.h"
 #include "DolphinQt/QtUtils/WrapInScrollArea.h"
+#include "DolphinQt/Settings.h"
 
 #include "VideoCommon/VideoBackendBase.h"
 #include "VideoCommon/VideoConfig.h"
 
+// List of graphics settings that are linked:
```

Comment:
I know you mentioned this but what is the intent of this list?

* Is it to help users adding new settings?
* Is it to direct people making UI edits?

I just question if this location helps either of those.  The only thought I have is adding these as comments for the UI that is being added.  But I'm not even sure that is reasonable, as such information can become dated quickly...

(As a somewhat contrived example, my PR for the post processing removal removed the last bullet, but what if when that gets merged, I or someone else forgets to remove this comment?)